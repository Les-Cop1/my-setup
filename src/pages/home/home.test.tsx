import React from 'react'

import { Home } from '@pages'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

describe('Home', () => {
  test('Snapshot', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
