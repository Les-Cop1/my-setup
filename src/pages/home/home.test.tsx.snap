// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot 1`] = `
<div>
  <div
    class="max-w-7xl mx-auto flex flex-col content-center lg:flex-row justify-between p-5 gap-6 my-auto"
  >
    <main
      class="w-full lg:w-1/2 mt-10 mx-auto px-4 sm:mt-12 sm:px-6 md:mt-16 lg:mt-20 lg:px-8 xl:mt-28"
    >
      <h1
        class="text-4xl tracking-tight font-bold text-slate-800 sm:text-5xl sm:tracking-tight md:text-6xl md:tracking-tight"
      >
        mySetup
      </h1>
      <p
        class="mt-3 text-base text-slate-500 sm:mt-5 sm:text-lg sm:max-w-xl sm:mx-auto md:mt-5 md:text-xl lg:mx-0"
      >
        My setup presentation
      </p>
    </main>
    <img
      alt="chart illustration"
      class="w-full lg:w-1/2"
      src=""
    />
  </div>
</div>
`;
